setup.py

```diff
@@ -26,7 +26,8 @@
     packages=find_packages(exclude=("lookout.style.format.tests",)),
     namespace_packages=["lookout"],
     entry_points={
-        "console_scripts": ["analyzer=lookout.__main__:main"],
+        "console_scripts": ["analyzer=lookout.__main__:main",
+                            "format-analyzer=lookout.style.format.__main__:main"],
```

Comment:
Let's add `analyzer tool lookout.style.format --whatever` which delegates to `lookout.style.format.__main__`.
I feel uncomfortable with polluting the PATH.